src/devices/cpu/drcbex86.cpp

```diff
@@ -2387,7 +2387,7 @@ void drcbe_x86::op_nop(Assembler &a, const instruction &inst)
 
 void drcbe_x86::op_break(Assembler &a, const instruction &inst)
 {
-	a.int3();
+	a.call(imm(&osd_break_into_debugger));
```

Comment:
Do you need to store a pointer to a string at (%esp) for the argument to `osd_break_into_debugger`?